wikibaseintegrator/entities/item.py

```diff
@@ -36,6 +38,19 @@ def new(self, **kwargs) -> Item:
         return Item(self.api, **kwargs)
 
     def get(self, entity_id, **kwargs) -> Item:
+        if isinstance(entity_id, str):
+            pattern = re.compile(r'^Q?([0-9]+)$')
+            matches = pattern.match(entity_id)
+
+            if not matches:
+                raise ValueError("Invalid item ID ({}), format must be 'Q[0-9]+'".format(entity_id))

```

Comment:
Nice that you state the value it hiccuped on :)

Reply:
The user can now use 42, '42' and 'Q42' as item id.